math_notes/mathnotes.py

```diff
@@ -0,0 +1,208 @@
+from pathlib import Path
+
+import csv
```

Comment:
Nobody really uses this package anymore especially in data science. Pandas is better for working with tabular data, including with csv.